CHANGELOG.md

```diff
@@ -67,14 +92,14 @@
 - Merge pull request [#253](https://github.com/kubewarden/kubewarden-controller/issues/253) from kubewarden/road-to-1.1.0
 
 
-<a name="v1.0.0"></a>
-## [v1.0.0] - 2022-06-15
-
 <a name="v1.0.0-rc3"></a>
 ## [v1.0.0-rc3] - 2022-06-15
 
 <a name="v1.0.0-rc4"></a>
 ## [v1.0.0-rc4] - 2022-06-15
+
+<a name="v1.0.0"></a>
+## [v1.0.0] - 2022-06-15
```

Comment:
git-chglog swaps the order of v1.0.0 and its rcs. What the heck. I can't even. Anyway.